docs/sources/setup-grafana/configure-security/configure-authentication/github/index.md

```diff
@@ -46,140 +71,147 @@ scopes = user:email,read:org
+| `role_attribute_path`        | No       | [JMESPath](http://jmespath.org/examples.html) expression to use for Grafana role lookup.                                                                 |                                               |
+| `role_attribute_strict`      | No       | Set to `true` to deny user login if Grafana role cannot be extracted using `role_attribute_path`.                                                        | `false`                                       |
+| `allow_assign_grafana_admin` | No       | Set to `true` to enable automatic sync of Grafana server administrator role.                                                                             | `false`                                       |
+| `skip_org_role_sync`         | No       | Set to `true` to stop automatically syncing user roles.                                                                                                  | `false`                                       |
+| `allowed_organizations`      | No       | List of comma- or space-separated organizations. User should be a member of at least one organization to log in.                                         |                                               |
+| `allowed_domains`            | No       | List of comma- or space-separated domains. User should belong to at least one domain to log in.                                                          |                                               |
+| `team_ids`                   | No       | String list of team IDs. If set, user has to be a member of one of the given teams to log in.                                                            |                                               |
+| `tls_skip_verify_insecure`   | No       | If set to `true`, client will not verify the server's certificate chain and host name. Should only be set to `true` for testing purposes.                | `false`                                       |
+| `tls_client_cert`            | No       | Path of the certificate.                                                                                                                                 |                                               |
+| `tls_client_key`             | No       | Path to the key.                                                                                                                                         |                                               |
+| `tls_client_ca`              | No       | Path to the trusted certificate authority list.                                                                                                          |                                               |
 
-Restart the Grafana back-end. You should now see a GitHub login button
-on the login page. You can now login or sign up with your GitHub
-accounts.
+### Configure automatic login
```

Comment:
I'm confused about this and the next 6 sections. Are they there to provide more description/definition for certain configuration parameters defined in the table above?

Reply:
Yes, exactly.